### DISCUSSION TITLE
Can we check whether font fille (.ttf)  is downloaded in network tab to ensure font icons loaded or not

### DISCUSSION THREAD

**Discussion by devikabnh** [getgauge/taiko#2633] [General]
Can we check whether font fille (.ttf)  is downloaded in network tab to ensure font icons loaded or not

**Comment by zabil** [comment]
I haven't tried this but you should be able to use [client](https://docs.taiko.dev/api/client/) which is a wrapper over [CDP](https://chromedevtools.github.io/devtools-protocol/) to check for [Network](https://chromedevtools.github.io/devtools-protocol/tot/Network/) tab.
